### PR TITLE
[NO TESTS NEEDED] Rename podman machine create to init and clean up

### PR DESCRIPTION
**Pull request by ashley-cui**
Rename podman machine create to init because we're initing a VM, not
really creating it
Wire up CPUs flag
Suppress QEMU GUI from popping up

Signed-off-by: Ashley Cui <acui@redhat.com>

### CODE REVIEW

**Comment by ashley-cui** [comment]
@baude PTAL: what happened here is 

1. rename create -> init
2. wired CPUs flag
3. QEMU GUI doesn't pop up anymore
4. Some small docs change and removal of a debug statement

File: pkg/machine/qemu/machine.go [hunk 1]
```diff
@@ -88,8 +88,9 @@ func NewMachine(opts machine.CreateOptions) (machine.VM, error) {
 
 	// Add network
 	cmd = append(cmd, "-nic", "user,model=virtio,hostfwd=tcp::"+strconv.Itoa(vm.Port)+"-:22")
+	// Disable graphic window
+	cmd = append(cmd, "-display", "none")
```
Comment (baude) [inline pkg/machine/qemu/machine.go:92]: can make this condition on not being in log-level=debug?

Comment (ashley-cui) [reply pkg/machine/qemu/machine.go:92]: Sure, I'll look into that

**Comment by baude** [comment]
LGTM

**Pull request merged**